k8s/aat/common/rpe/send-letter-service.yaml

```diff
@@ -11,6 +11,8 @@ spec:
   releaseName: rpe-send-letter-service
   rollback:
     enable: true
+    retry: true
+    maxRetries: 5
```

Comment:
This should be default ?
```suggestion
```

Reply:
it's meant for teams to know that the option exists ... but, on second thought, do we want them to know?